DOC-2446: New `revisionhistory_fetch_revision` option to support updating a revision when it is selected. (#3349)
tinymce/tinymce-docs: modules/ROOT/pages/7.2-release-notes.adoc

For information on the **Image Editing** plugin, see: xref:editimage.adoc[Image Editing].

=== Revision History

The {productname} {release-version} release includes an accompanying release of the **Revision History** premium plugin.

This release of **Revision History** introduces the following addition.

==== New `revisionhistory_fetch_revision` option to support updating a revision.
// #TINY-10362

In {productname} {release-version}, a new `revisionhistory_fetch_revision` option has been added to enable updating a revision after the initial fetch. If configured, when a revision is selected, the plugin uses this option to update the selected or closest revision that has no content.

> NOTE: As part of this addition, the `content` property is now optional. For the fetched revisions, empty string is considered as valid content.

Reviewer: ```suggestion
NOTE: As part of this addition, the `content` property is now optional. For the fetched revisions, an empty string is considered as valid content.
```

> NOTE: The integrators still need to configure the `revisionhistory_fetch` option to provide the initial revisions.

Reviewer: ```suggestion
IMPORTANT: The `revisionhistory_fetch` option must be configured when using the new `revisionhistory_fetch_revision` option, as it is required to provide the initial revisions.
```

For more information on the `revisionhistory_fetch_revision` option, see: xref:revisionhistory.adoc#revisionhistory_fetch_revision[Revision History - revisionhistory_fetch_revision].

> This option would be helpful when dealing with revisions that have large content or preferring to only load the content when required.

Reviewer: ```suggestion
This option is useful for handling revisions with large content or for loading content only when needed.
```
